index.html

```diff
@@ -194,49 +198,228 @@ <h2>
+                  object</a>'s <a data-cite=
+                    "!HTML52/webappapis.html#responsible-browsing-context">responsible
+                    browsing context</a>'s [[!HTML52]] <a data-cite=
+                      "!HTML52/browsers.html#top-level-browsing-context">top-level
+                      browsing context</a>'s <a data-cite=
+                        "!HTML52/browsers.html#active-document">active document</a>'s
+                      origin.</p>
+              </li>
+              <li>
+                <p>If the <a>current settings object</a>'s origin is
+                different from <var>originIdentifier</var>, set
+                <var>originIdentifier</var> to the result of combining
+                <var>originIdentifier</var> and the <a>current settings
+                  object</a>'s origin.</p>
+              </li>
+              <li>
+                <p>Let <var>p</var> be a new promise.</p>
+              </li>
+              <li>
+                <p>Run the following steps in parallel:</p>
+                <ol>
+                  <li>
+                    <p>For each media type <var>T</var> in
+                    <var>requestedMediaTypes</var>,</p>
+                    <ol>
+                      <li>
+                        <p>If no sources of type <var>T</var> are available,
+                        <a>reject</a> <var>p</var> with a new
+                        <code><a>DOMException</a></code> object whose
+                        <code><a>name</a></code> attribute has the value
+                        <code>NotFoundError</code>.</p>
+                      </li>
+                      <li>
+                        <p><a>Retrieve the permission state</a> for obtaining
+                        sources of type <var>T</var> in the current browsing
+                        context. If the permission state is "denied", jump to
+                        the step labeled <em>PermissionFailure</em> below.</p>
+                      </li>
+                    </ol>
+                  </li>
+                  <li>
+                    <p>Optionally, e.g., based on a previously-established
+                    user preference, for security reasons, or due to platform
+                    limitations, jump to the step labeled <em>Permission
+                      Failure</em> below.</p>
+                  </li>
+                  <li>
+                    <p>For the origin identified by
+                    <var>originIdentifier</var>, <a>prompt the user to choose</a>
+                    a display device, with a PermissionDescriptor named
+                    <code>"display"</code>, resulting in a set of
+                    provided media.</p>
+                    <p>The provided media MUST include precisely one track of
+                    each media type in <var>requestedMediaTypes</var>.
+                    The devices chosen MUST be the ones determined by the user.
```

Comment:
nit: Language like "devices" should be updated, we're talking about display surfaces.